app/util/process_utils.py

```diff
@@ -23,3 +25,26 @@ def kill_gracefully(process, timeout=2):
         stdout, stderr = process.communicate()
 
     return process.returncode, stdout, stderr
+
+
+def Popen_with_delayed_expansion(cmd, *args, **kwargs):
```

Comment:
@josephharrington I don't like to name it `Popen` as it's confusing to people. Why I should use this `Popen` instead of `subprocess.Popen`? Naming it this way, IMO, is explicit so hopefully more readable and maintainable. I am going to merge this as is but I am more than happy to change it later. :smiley: